c-sharp-syllabus-main/c-sharp-syllabus-main/csharp-basics/exercises/ClassesAndObjects/Exercise 3/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise_3
+{
+    internal class FuelGauge
+    {
+        public double _currentFuel;
+        public FuelGauge(double currentFuel)
+        {
+            this._currentFuel = currentFuel;
+        }
+        public double StartFuel()
+            {
+            Console.WriteLine("Enter yout current fuel amount in liters");
+            double currentFuel = double.Parse(Console.ReadLine());
+            this._currentFuel = currentFuel;
+            return currentFuel;
+            }
+        public void Increment(double currentFuel)
+        {
+            if (currentFuel < 70)
+            {
+                this._currentFuel = 70;
+            }
+            else
+            {
+                Console.WriteLine("Gas tank is full. You must drive!");
+                this._currentFuel = 70;
```

Comment:
Šisbūs ja ir 70 un vairāk?

Reply:
Tā kā bākas max ietiltība ir 70l, tad, ja lies vairāk, to nebūs iespējams izdarīt un bākā tāpat būs 70l. Arī tad, ja lies pāri malām :)